forms-html/yara-abdulrahman/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+<title> HTML FORM</title>
+    </head>
+
+    <body>
+       
+        <form action="#" target="_self" autocomplete="on">
+        <label for="email">Enter your email:</label>
+        <input type="email" id="email" name="email" required>
+     
+        <br>
+        <label for="username">Username:</label>
+        <input type="text" id="username" name="username">
+        <br>
+        <label for="passworrd">Password:</label>
+        <input type="password" id="password" name="password">
+        <br>
+        <label for="quantity">Quantity (between 1 and 5)</label>
+        <input type="number" id="quantity" name="quantity" max="5" min="1"> 
+        <br>
+    <label for="volume">Volume (Between 0 and 50)</label>
+    <input type="range" id="volume" name="volume" min="0" max="50" value="25">
+    <br>
+     <h4>Radio</h4>
+        <input type="radio" id="male "value="male" name="Gender">
+        <label for="male">Male</label><br>
+        <input type="radio" id="female "value="female" name="Gender">
+        <label for="female">Female</label><br>
+        <input type="radio" id="other "value="other" name="Gender">
```

Comment:
don't make spaces inside IDs value `id="male "` to `id="male"`
fix here and all.